docs/create-webhook.yaml

```diff
@@ -135,6 +168,61 @@ spec:
       then
         exit 0
       fi
+      export EVENTBODY="(inputs.params.EventBody)"
+      export EVENTHEADERS="(inputs.params.EventHeaders)"
+      export GITHUBSECRET="(inputs.params.Github-Secret)"
+      export GITHUBSECRETKEY="(inputs.params.Github-Secret-Key)"
+      export D="\$"
+      cat <<EOF | kubectl apply -f -
+      apiVersion: tekton.dev/v1alpha1
+      kind: Task
+      metadata:
+        name: $(inputs.params.ValidateTaskName)
+      spec:
+        inputs:
+          params:
+            - name: EventBody
+              description: Payload of Event Received
+            - name: EventHeaders
+              description: Headers of the Event Received
+            - name: Github-Secret
+              description: Secret name used to configure webhook
+            - name: Github-Secret-Key
+              description: Secret key used to configured webhook
+        steps:
+          - name: validate
+            image: python:alpine
+            command: ["/bin/sh"]
+            args:
+            - -ce
+            - |
+              set -e
+              cat <<EOF | python
+              import hashlib, os, hmac, json
+              secret = bytes(os.environ.get('GithubSecret'), 'utf-8')
+              payload = bytes('${D}${EVENTBODY}','utf-8')
+              h = json.loads(r'${D}${EVENTHEADERS}')
+              signature = h["X-Hub-Signature"][0]
+              expected = hmac.new(secret, payload, hashlib.sha1).hexdigest()
+              if expected is not None:
+                if not isinstance(expected, str):
+                  expected = str(expected)
+                sig_parts = signature.split("=", 1)
+                if not isinstance(sig_parts[1], str):
+                  sig_parts1 = str(sig_parts[1])
+                else:
+                  sig_parts1 = sig_parts[1]
+                if len(sig_parts) > 1 and sig_parts[0] == "sha1" and hmac.compare_digest(sig_parts1, expected):
+                  exit(0)
+              exit(1)
+              EOF
+            env:
+              - name: GithubSecret
+                valueFrom:
+                  secretKeyRef:
+                    name: ${D}${GITHUBSECRET}
+                    key: ${D}${GITHUBSECRETKEY}
+      EOF
```

Comment:
I don't really like the idea of copying and pasting the [validate-github-event](https://github.com/tektoncd/triggers/blob/master/docs/validate-github-event.yaml) into this yaml file, because if we have to update the Task, then we'll have to modify two places. 

Also, I don't know if this is necessary. Just like a user will have to `kubectl apply -f create-webhook.yaml`, before using the `create-webhook` Task, they can also `kubectl apply -f validate-github-event.yaml` before using the `validate-github-event` Task.